redisenterprise/manifest.json

```
@@ -0,0 +1,41 @@
+{
+  "display_name": "Redis Enterprise",
+  "maintainer": "github@mague.com",
+  "manifest_version": "1.0.0",
+  "name": "redisenterprise",
+  "metric_prefix": "redisenterprise.",
+  "metric_to_check": "redisenterprise.total_node_count",
+  "creates_events": true,
+  "short_description": "Redis Enterprise Observability",
+  "guid": "727dcbe6-9ed6-409f-ad72-265939b90da8",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "RedisEnterprise Integration",
+  "categories": [
+    "data store",
+    "caching",
+    "log collection"
```

```suggestion
    "caching"
```
`log collection` is reserver for integration that tail log files and send them to the logs product.